cardsavr/Aes256.cs

```diff
@@ -94,38 +87,25 @@ public string Decrypt(string b64CipherText, string b64Iv, string b64Key)
         /// </summary>
         public string Encrypt(string clearText, string b64Key)
         {
-            byte[] iv = _testIv ?? GetRandomBytes(16);
-            byte[] key = Convert.FromBase64String(b64Key);
+            var aesGcm = new System.Security.Cryptography.AesGcm(Convert.FromBase64String(b64Key));
```

Comment:
This doesn't hurt but you don't have to use the fully qualified name since System.Security.Cryptography has already been declared at the top.
Same on line 94.

Reply:
Updated.